src/redux/Courier/taskSelectors.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Selectors
+ *
+ * Selectors help decouple the shape of the state from the component code itself.
+ * Here we use `reselect`, which allows us to memoize computed property values,
+ * benefitting performance.
+ */
+import { createSelector } from 'reselect'
+
+export const selectTaskSelectedDate = state => state.ui.tasks.selectedDate
+export const selectTriggerTasksNotification = state => state.entities.tasks.triggerTasksNotification
+export const selectIsTasksLoading = state => state.entities.tasks.isFetching
+export const selectIsTasksLoadingFailure = state => state.entities.tasks.fetchError
+export const selectTasks = state => state.entities.tasks
```

Comment:
Should be `state.entities.tasks.items` no ? Otherwise `selectTasksList` won't work